### ISSUE TITLE
USER STORY: Model Optimization V3

### ISSUE THREAD

**Issue by Huddy2022** [Huddy2022/milestone-project-mildew-detection-in-cherry-leaves#22]
As a **data scientist** I want to **train the CNN model using the pre-processed dataset with re-sized images 75 x 75 and optimize the hyperparameters min value 128, max value 256, step 32, filters 32, 64 & 64, dropout 0.5, with values 1e-3, 1e-4 on a Relu activation and loss of binary_crossentropy to achieve the best performance,** so that **the model can accurately classify cherry leaves as healthy or containing powdery mildew.**
